server/controllers/interview.js

```diff
@@ -0,0 +1,109 @@
+// connect to the Interview Model
+let Interview = require('../models/interview');
+let User = require('../models/user');
+
+module.exports.createInterview = async (req, res, next) => {
+	const { guest, date, theme, questions, difficult } = req.body;
+	const user = await User.findOne({ email: req.user.user }, 'id');
+
+	let newInterview = Interview({
+		owner: user,
+		guest: guest,
+		date: date,
+		theme: theme,
+		questions: questions,
+		difficult: difficult,
+	});
+
+	Interview.create(newInterview, (err) => {
```

Comment:
You're mixing create methods here, I think this would technically work but Interview.create expects an object or array rather than an Interview instance. You can use newInterview.save() for an instance.

Reply:
The method has been changed as requested